settings.gradle.kts

```diff
@@ -41,6 +44,9 @@ include(":runtime:protocol:http")
 include(":runtime:protocol:http-client-engines:http-client-engine-ktor")
 
 include(":compile-tests")
+include(":benchmarks")
```

Comment:
question

given the note above about the module failing to compile until `assemble` is called, perhaps we want to gate the inclusion of these modules behind some kind of `local.properties` setting?  Making it opt-in essentially.  Do we expect there to be actions performed in these modules as part of routine SDK development?

Reply:
What note are you referring to? The module should compile fine out the gate.